packages/metals-languageclient/src/getJavaHome.ts

```diff
@@ -35,14 +35,12 @@ async function validateJavaVersion(
     const javaVersionOut = spawn(javaBin, ["-version"], {
       encoding: "utf8",
     });
+    const javaInfoStr = (await javaVersionOut).stderr as string;
 
-    javaVersionOut.stderr?.on("data", (out: Buffer) => {
-      outputChannel.appendLine(`${javaBin} -version:`);
-      const msg = "\t" + out.toString().trim().split("\n").join("\n\t");
-      outputChannel.appendLine(msg);
-    });
+    const msg = "\t" + javaInfoStr.toString().trim().split("\n").join("\n\t");
+    outputChannel.appendLine(`${javaBin} -version:`);
+    outputChannel.appendLine(msg);
```

Comment:
I think the reason we had `javaVersionOut.stderr?.on("data", ...)` is because, if running `java -version` fails `await javaVersionOut` will throw, so `(await javaVersionOut).stderr as string` won't get printed.

Reply:
Ok, good point. I will just remove the initial line with `-version` there. It's not really adding anything and we already print which java is being looked into before that